### DISCUSSION TITLE
Is threre a way to install a MeterFilter for gql metrics?

### DISCUSSION THREAD

**Discussion by hzariv** [Netflix/dgs-framework#1986] [Q&A]
I would like to install MeterFilter for out of the box metrics described [here](https://netflix.github.io/dgs/advanced/instrumentation/#metrics-out-of-the-box) but I not able to get the MeterRegistry that DGS framework is using. According to this [autoconfig](https://github.com/briandilley/dgs-framework/blob/master/graphql-dgs-spring-boot-micrometer/src/main/kotlin/com/netflix/graphql/dgs/metrics/micrometer/DgsGraphQLMicrometerAutoConfiguration.kt#L94) class, DgsMeterRegistrySupplier is internal. Is there a way to get the hold of MeterRegistry instance that DGS framework is using for gql metrics?

I am currently using DGS version 5.4.3

**Comment by srinivasankavitha** [comment]
What are you looking to achieve with the filter in this case? We do have a tags contributor if you want to contribute tags to existing metrics: https://netflix.github.io/dgs/advanced/instrumentation/#further-tag-customization

**Reply by hzariv** [reply]
No actually this is a work around for this enhancement I filed: https://github.com/Netflix/dgs-framework/issues/1974
We are facing a high cardinality due to loaderBatchSize tag in dataloader metrics. So since I don't have a way to limit the cardinality for this tag, I am planning to ignore it.

`meterRegistry.config().meterFilter(MeterFilter.ignoreTags("gql_loaderBatchSize"));`

TagCustomizer API seems to only allow adding new tags as I understand it.
